docs/setup-using-puppet.md

```diff
@@ -0,0 +1,84 @@
+# Setup Minio using Puppet [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io)
+
+[Puppet][puppet] is an Open Source configuration management system, allowing for
+automated system provisioning.
+
+In this recipe you will learn how to deploy Minio using Puppet as standalone
+installation.
+
+## 1. Prerequsites
+
+This cookbook assumes that you have a working Puppet setup (at least an agent for
+running Puppet recipes).
```

Comment:
A link to where reader can get instructions to setup puppet would be very essential here.